auth.py

```diff
@@ -0,0 +1,43 @@
+from fastapi.security import OAuth2PasswordBearer
+from fastapi import HTTPException
+from datetime import datetime, timedelta
+import jwt
+
+fake_users_db = {
+  "yamakenji": {
+      "id": "base64id",
+      "username": "yamakenji",
+      "full_name": "Kenji Yamashita",
+      "email": "sample@example.com",
+      "password": "fakehashedsecret",
+      "disabled": False,
+  }
+}
+# for instance 
+secret_key = '09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7'
```

Comment:
confirmed that using fake information for the time being.

Reply:
[REQUEST CHANGES]

I think it is necessary to change the method of reading **the secret key** from the file.
It is also better to read the location of **the secret key file** from the environment variable.